framework/src/main/java/org/checkerframework/common/basetype/BaseTypeChecker.java

```diff
@@ -584,6 +587,26 @@ private void printStoredMessages(CompilationUnitTree unit) {
         if (messageStore != null) {
             for (CheckerMessage msg : messageStore) {
                 super.printOrStoreMessage(msg.kind, msg.message, msg.source, unit);
+                this.printStackTrace(msg.trace);
+            }
+        }
+    }
+
+    /**
+     * Prints given stack trace of runtime exception if -AdumpOnErrors flag is passed as compilation
+     * argument
+     *
+     * @param trace stack trace when the checker encounters an error
+     */
+    private void printStackTrace(StackTraceElement[] trace) {
+        boolean dumpOnErrors =
+                (processingEnv != null
+                        && processingEnv.getOptions() != null
+                        && processingEnv.getOptions().containsKey("dumpOnErrors"));
+        if (dumpOnErrors) {
+            System.err.println(new RuntimeException().toString());
```

Comment:
You don't want this output, as it's a stacktrace in this same location for every error.

Reply:
I have removed this statement in my latest commit